src/css/components/_control.scss

```diff
@@ -36,3 +36,28 @@
   display: table-cell;
   vertical-align: middle;
 }
+
+// visually hide a control-bar component without changing the position
+// and without hiding it from screen readers. This also preserves
+// a transition animation.
+.video-js .vjs-control .vjs-visual-hide-vertical {
```

Comment:
yeah I originally had a setTImeout call on `hide()` that would run after `0.4s` that made the 1px go away but it made things a lot more complex. Hopefully we can find another fix for this. Maybe this should go with the IE8/IE9 style fix PR?